interfaces/builtin/serial_port.go

```diff
@@ -154,7 +156,7 @@ func (iface *SerialPortInterface) ConnectedPlugSnippet(plug *interfaces.Plug, sl
 		if iface.hasUsbAttrs(slot) {
 			// This apparmor rule must match serialDeviceNodePattern
 			// UDev tagging and device cgroups will restrict down to the specific device
-			return []byte("/dev/tty* rw,\n"), nil
+			return []byte("/dev/tty[A-Z]*[0-9] rw,\n"), nil
```

Comment:
@jdstrand can you comment whether that ^ is supported in the AppArmor globbing syntax? I don't see a '+' mentioned in the project wiki page.